MVVMIOS/ViewModel/Movie/MovieViewModel.swift

```diff
@@ -71,19 +73,23 @@ class MovieViewModel {
             case .success(let data):
                 print(data)
             case .failure(let error):
-                self.viewStateFor(for: error)
+                self.resultPresentable?.setupViewState(self.viewStateFor(for: error))
```

Comment:
Should consider whether you want to have `self` captured strongly or not in this closure. Does the ViewController hold the ViewModel strongly? If so then the only way the VM can be gone is if the VC is also gone, and if that's the case why bother updating the view state? But that all depends on who holds the VM strongly